nima/http2/webserver/src/main/java/io/helidon/nima/http2/webserver/Http2ConfigBlueprint.java

```diff
@@ -117,6 +118,31 @@ public interface Http2Config {
      *
      * @return settings for computing the requested URI
      */
-    @ConfiguredOption(key = "requested-uri-discovery")
+    @ConfiguredOption
     RequestedUriDiscoveryContext requestedUriDiscovery();
+
+    /**
+     * Protocol configuration type.
+     *
+     * @return type of this configuration
+     */
+    default String type() {
+        return Http2ConnectionProvider.CONFIG_NAME;
+    }
+
+    class Http2ConfigInterceptor implements Prototype.BuilderInterceptor<Http2Config.BuilderBase<?, ?>> {
+        @Override
+        public Http2Config.BuilderBase<?, ?> intercept(Http2Config.BuilderBase<?, ?> target) {
+            if (target.name() == null) {
+                target.name("@default");
```

Comment:
@default was a const somewhere in common - we should probably use that everywhere instead of this literal.

Reply:
Follow up created